src/main/handlers/misc/reportBug.ts

```diff
@@ -0,0 +1,26 @@
+import fetch from 'node-fetch';
+
+const reportBug = async (title: string, body: string) => {
+  try {
+    await fetch(
+      'https://mlvet-bug-reporter-mlvet-bug-reporter-9xw6.vercel.app/api/reportBug',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          // eslint-disable-next-line object-shorthand
+          title: title,
+          description: body,
+        }),
+      }
+    );
+    return 200;
+  } catch (e) {
+    console.error(e);
+    return -1;
```

Comment:
what's this about?

Reply:
Looks like you're treating it as a status code? -1 is definitely not an HTTP status code. If you want to go this route of returning a status code then return the actual status code that was returned

Reply:
fixed on server code